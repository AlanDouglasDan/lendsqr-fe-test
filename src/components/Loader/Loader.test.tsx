import { BrowserRouter } from "react-router-dom";
import { render } from "@testing-library/react";

import Loader from "./Loader";

describe("Loader Component Test", () => {
  it("Loader Component Snapshot", () => {
    const { asFragment } = render(
      <BrowserRouter>
        <Loader />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
