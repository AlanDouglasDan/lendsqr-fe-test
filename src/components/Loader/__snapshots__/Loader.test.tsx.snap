// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component Test Loader Component Snapshot 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
  >
    <div
      class="lds-roller"
    >
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</DocumentFragment>
`;
